CMakeLists.txt

```diff
@@ -255,6 +260,7 @@ elseif (CMAKE_C_COMPILER_ID STREQUAL "MSVC")
     # using Visual Studio C/C++
     set(BASE_RELEASE_FLAGS "${CMAKE_C_FLAGS} /DLIBPLCTAGDLL_EXPORTS=1 /W3")
     set(BASE_DEBUG_FLAGS "${CMAKE_C_FLAGS} /DLIBPLCTAGDLL_EXPORTS=1 /W3")
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
```

Comment:
IIRC, this will cause the compiler to not throw warnings about some older standard C functions.  I have avoided that in the core library by using the Windows versions (blah_r or blah_s usually) of these functions.   I would prefer to keep the warnings on.   Microsoft does change what things warn and what things just do not compile from version to version, so I think this may be a dangerous change.   Can you try to make it compile on Windows without this?  Fine if you cannot, but it would be nice if it could.